modules/conflict_resolver/php/NDB_Menu_Filter_resolved_conflicts.class.inc

```diff
@@ -113,21 +112,27 @@ class NDB_Menu_Filter_Resolved_Conflicts extends NDB_Menu_Filter
         );
 
         $this->headers = array(
-                          'TableName',
+                          'Instrument',
                           'CandID',
                           'PSCID',
                           'Visit_label',
-                          'ResolutionTimestamp',
+                          'Timestamp',
                          );
 
         if ($useProjects) {
             $this->headers = array_merge(
                 $this->headers,
-                array('candidate.ProjectID' => 'Project')
+                array('Project')
             );
         }
-
-        $this->headers[] = 'FieldName';
+        $this->headers =array_merge(
+            $this->headers,
+            array(
+             'Question',
+             'Correct_Answer',
+            )
+        );
+        $this->tpl_data['hiddenHeaders'] = json_encode(['Timestamp']);
```

Comment:
The timestamp shouldn't be hidden. That's how it was before
![image](https://cloud.githubusercontent.com/assets/6627543/19355330/fbc055ba-9137-11e6-9245-765e18d2a9b8.png)